### ISSUE TITLE
Question - Is it possible to simply not use a secrets provider?

### ISSUE THREAD

**Issue by adrians5j** [pulumi/pulumi#7140]
If I'm not mistaken, at the very least, users today have to use the "passphrase" secrets provider.

But, let's say I'm not planning on storing sensitive secrets stored in my `Pulumi.*.yaml`, would it be possible to just ignore this option? 

It's not anything urgent nor super important. I just wanted to hear your thoughts on the matter.

Thank you!

**Comment by lukehoban** [comment]
Indeed, there is no option today to use no secrets provider. It is something that could definitely be added in the future. 

Likely the easiest way to support this would be to add support for a no-op seecrets provider that just stored plaintext values as its ciphertext.

**Comment by pierskarsenbarg** [comment]
One of the core principles for Pulumi is that it's secure by default. 

You can set an environment variable `PULUMI_CONFIG_PASSPHRASE` to be the passphrase so you don't have to type it in if that helps?
